src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,35 @@
+<div class="footer">
+  <div class="contact">
+    <div class="wrapper logo-wrapper">
+      <img src="../../../img/footer.png" alt="" />
```

Comment:
```suggestion
      <img src="../../../img/footer.png" alt="Switch" />
```
画像で文字を表現している際は、その内容を `alt` 属性に入れておきましょう！